modules/neo4j/src/main/java/org/testcontainers/containers/Neo4jContainer.java

```diff
@@ -205,6 +205,10 @@ public S withoutAuthentication() {
      * @return This container.
      */
     public S withDatabase(MountableFile graphDb) {
+        if (!usesVersion("3")) {
```

Comment:
I hope this will not end up like a Windows 98 check in 20 years (starts with 9😆).

Reply:
Pragmatic checking. Already improved by two more characters in the string :wink: